cmd/kion/console/console.go

```diff
@@ -49,31 +50,52 @@ func run(cfg *config.Config, keyCfg *config.KeyConfig) error {
 	if err != nil {
 		return err
 	}
-	region, err := cfg.StringErr("region")
+
+	kion, err := util.NewClient(cfg, keyCfg)
 	if err != nil {
 		return err
 	}
 
-	kion, err := util.NewClient(cfg, keyCfg)
+	accountInfo, err := kion.GetAccountByID(accountID)
 	if err != nil {
 		return err
 	}
+
+	awsEndpoint, region := "", ""
+
+	// Account types: 1=commercial, 2=govcloud
```

Comment:
use constants for account types and aws endpoints?

Reply:
IMO, since the string literals for the AWS domains are only in one place, constants aren't needed.